dist/docker/scylla-manager-agent.dockerfile

```diff
@@ -1,15 +1,13 @@
-FROM ubuntu
-ARG ARCH=amd64
+FROM docker.io/redhat/ubi9-minimal:latest
```

Comment:
Should have used quay.io really.

It's unclear why we use latest, but we can review this decision later.

Reply:
In fact, we should use registry.access.redhat.com/ubi9/ubi:latest

Reply:
This was not addressed. Please do.